onnx/checker.cc

```diff
@@ -253,7 +262,21 @@ void check_model(const ModelProto& model, int ir_version) {
       }
     }
   }
-  check_graph(model.graph(), model.ir_version());
+  std::unordered_map<std::string, int> versions;
+  CheckerContext ctx;
+  ctx.ir_version = model.ir_version();
+  for (const auto& opset_import : model.opset_import()) {
+    ctx.opset_imports[opset_import.domain()] = opset_import.version();
+  }
+  auto dit = ctx.opset_imports.find("");
```

Comment:
how about we add one const string "ONNX_DOMAIN", which may be a little bit more readable please?

Reply:
ok